core/src/test/java/org/hisp/dhis/android/core/arch/api/filters/internal/FilterConverterShould.java

```diff
@@ -141,58 +141,6 @@ public void returns_correct_path_when_create_a_retrofit_request_using_filters_an
                 "/api?field=id,code,name,displayName&filter=id:in:[uid1,uid2]&filter=lastUpdated:gt:updatedDate");
     }
 
-    @Test
-    public void returns_correct_path_when_create_a_retrofit_request_ignoring_null_filter() throws IOException, InterruptedException {
-        ArrayList<String> values = new ArrayList<>(2);
-        values.add("uid1");
-        values.add("uid2");
-
-        TestService service = testService(server);
-
-        service.test(
-                InFilter.create(Field.create("id"), values),
-                SingleValueFilter.gt(Field.create("lastUpdated"), null)
-        ).execute();
-
-        RecordedRequest request = server.takeRequest();
-
-        assertThat(request.getPath()).isEqualTo("/api?filter=id:in:[uid1,uid2]");
-    }
-
-    @Test
-    public void returns_correct_path_when_create_a_retrofit_request_ignoring_empty_string_filter() throws IOException, InterruptedException {
-        ArrayList<String> values = new ArrayList<>(2);
-        values.add("uid1");
-        values.add("uid2");
-
-        TestService service = testService(server);
-
-        service.test(
-                InFilter.create(Field.create("id"), values),
-                SingleValueFilter.gt(Field.create("lastUpdated"), "")
```

Comment:
Maybe we have to give it a second thought because an empty string in a SingleValueFilter will lead to an invalid url. For example, this request throws an Internal Server Error `api/dataElements?filter=id:eq:&fields=id,name`.